pallets/capacity/src/lib.rs

```diff
@@ -212,6 +249,36 @@ pub mod pallet {
 	pub type UnstakeUnlocks<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, UnlockChunkList<T>>;
 
+	/// stores how many times an account has retargeted, and when it last retargeted.
+	#[pallet::storage]
+	#[pallet::getter(fn get_retargets_for)]
+	pub type Retargets<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, RetargetInfo<T>>;
+
+	/// Information about the current reward era. Checked every block.
+	#[pallet::storage]
+	#[pallet::whitelist_storage]
+	#[pallet::getter(fn get_current_era)]
```

Comment:
The `pallet::getter` macro is being deprecated I suggest using the alternative way `SomeStorage::<T>::get()`

Reply:
I will open a PR to remove these getters